WalletWasabi/Stores/BitcoinStore.cs

```diff
@@ -53,10 +55,4 @@ public async Task InitializeAsync(CancellationToken cancel = default)
 			await Task.WhenAll(initTasks).ConfigureAwait(false);
 		}
 	}
-
-	public async ValueTask DisposeAsync()
```

Comment:
I'm not sure you'll like this but it's certainly a way to fix more CA2000 warnings. Alternatively, we can just suppress them.

However, the current approach makes it hard to do this mistake: https://github.com/zkSNACKs/WalletWasabi/pull/7076/files#diff-4de23fa5cf841d63dc319053d5a1d29c903d9ee2c177fe2f445832ccccf7a703L60